src/config.zig

```diff
@@ -49,22 +49,22 @@ pub const memory_size_max_default = 1024 * 1024 * 1024;
 
 /// The maximum number of accounts to store in memory:
 /// This impacts the amount of memory allocated at initialization by the server.
-pub const accounts_max = switch (deployment_environment) {
+pub const cache_accounts_max = switch (deployment_environment) {
     .production => 1_000_000,
     else => 100_000,
 };
 
 /// The maximum number of transfers to store in memory:
 /// This impacts the amount of memory allocated at initialization by the server.
 /// We allocate more capacity than the number of transfers for a safe hash table load factor.
-pub const transfers_max = switch (deployment_environment) {
+pub const cache_transfers_max = switch (deployment_environment) {
     .production => 100_000_000,
     else => 1_000_000,
 };
 
 /// The maximum number of two-phase transfers to store in memory:
 /// This impacts the amount of memory allocated at initialization by the server.
-pub const transfers_pending_max = transfers_max;
+pub const cache_transfers_pending_max = cache_transfers_max;
```

Comment:
These values could be too high. For example, 100m transfers is 12.8 GiB.

They'll eventually be runtime known, instead of constants, but for now what do you think of going with 100k accounts (12.8 MiB) and 1m transfers (128 MiB)?